easydmarc.com.setup.json

```diff
@@ -0,0 +1,80 @@
+{
+  "providerId": "easydmarc.com",
+  "providerName": "EasyDMARC",
+  "serviceId": "setup",
+  "serviceName": "EasyDMARC setup",
+  "version": 1,
+  "syncRedirectDomain": "easydmarc.com, app.easydmarc.com",
+  "logoUrl": "https://easydmarc.com/img/wordmark.svg",
+  "syncPubKeyDomain": "domainconnect.easydmarc.com",
+  "warnPhishing": true,
```

Comment:
The template is signed. Typically no phishing risk shall be present, or?

Reply:
removed